language-support/java/codegen/src/test/scala/com/digitalasset/daml/lf/codegen/backend/java/ChoiceMetadataFieldsSpec.scala

```diff
@@ -7,7 +7,7 @@ import com.daml.ledger.javaapi.data.Unit
 import com.daml.ledger.javaapi.data.codegen.ChoiceMetadata
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
-import retro.InterfaceRetro
+import ut.retro.InterfaceRetro
```

Comment:
@ray-roestenburg-da My initial thought was that this test now referring to the right package for generated sources was sufficient. I'll see if we have any existing tests for location of files and extend that, else I'll have to write a new suite